src/commands/help.ts

```diff
@@ -1,37 +1,88 @@
-import { MessageEmbedOptions } from 'discord.js';
+import { MessageEmbed, Message } from 'discord.js';
 import { CommandDef } from './command-def';
 import { COMMANDS } from './commands';
+import { oneLine, stripIndent } from 'common-tags';
+import { Config } from '../config/get-config';
+
+type helpFunctionArgs = {
+  message: Message;
+  config?: Config;
+  Command?: string;
+};
+
+function sendFullList({ message, config }: helpFunctionArgs): void {
+  const helpEmbed: MessageEmbed = new MessageEmbed()
+    .setColor('#0099FF')
+    .setTitle('Help Command')
+    .addField(
+      'Bot Information',
+      oneLine`Hello! I am a test bot created by bradtaniguchi and members of
+            FreeCodeCamp.org to experiment with the process of building a
+            Discord Bot. You can view my
+            [source code here](https://github.com/bradtaniguchi/discord-bot-test)`
+    )
+    .addField(
+      'List of Commands',
+      COMMANDS.sort((a, b) =>
+        a.prefix.toUpperCase() > b.prefix.toUpperCase() ? 1 : -1
+      )
+        .map((command, i) => {
+          if ((i + 1) % 4 === 0) return `\`${command.prefix}\n\``;
+          else return `\`${command.prefix}\``;
+        })
+        .join(' ') +
+        stripIndent`
+        \u200b
+        \u200b
+        For more information about each command, use \`${
+          (config as Config).PREFIX
+        } help <command>\``
+    )
+    .setFooter('I am not affiliated with FreeCodeCamp in any way.');
+  message.author.send(helpEmbed);
+  message.channel.send('Your help message is sent through DM');
```

Comment:
```suggestion
  message.channel.send('DM sent with the requested information');
```

Slight wording change. Keeps it a little cleaner than using the second-person perspective.